Release5.md

```diff
@@ -0,0 +1,159 @@
+* With the `openstack-resource-manager` a new day 2 operations tool has been added.
+  Furthermore an osism role for tuned to optimize system profiles is now present.
+* The [openstack-flavor-manager](https://github.com/osism/openstack-flavor-manager) is now able to create all standard, mandatory SCS flavors for you.
+* Scaphandre Prometheus Exporter has been added to export power consumption metrics more easily.
+* To optimize system profiles an osism role for tuned is now present.
+* Full support for air-gapped installation and operation of environments.
+* A migration script and guide for moving from R4 to R5 clusters is available.
+* Metering has been improved and a reference billing API implementation is available as technical preview.
+
+### SCS Developer focused improvements (Cloud-in-a-Box, testbed and k8s cluster management)
+
+* Documentation on testbed and Cloud-in-a-Box have been reworked.
+* Reflecting CiaB's usage as edge cloud appliance, it now receives more automated testing.
+
+### Project Infrastructure
+* zuul.scs.community now complements OSISM's existing zuul infrastructure and is used also
+  by the container layer to execute the CNCF e2e tests.
+* registry.scs.community has been migrated to a new IaaS location (documented in a blog
+  article) and is kept up-to-date now.
+
+## Upgrade/Migration notes
+
+* For the IaaS reference implementation, please refer to the [OSISM 6.0.0 Upgrade Notes](https://release.osism.tech/notes/6.0.0.html#upgrade-notes).
+
+## Removals
+
+* Please check the removals for OSISM in the [upstream removal notices](https://release.osism.tech/notes/6.0.0.html#removals).
+
+* The services `minio.services.osism.tech` and `harbor.services.osism.tech` are deprecated and will be turned of on October 20th, 2023.
+
+## Deprecations
+
+### Deprecations via OSISM
+
+For these please also refer to the [upstream deprecation notices](https://release.osism.tech/notes/6.0.0.html#deprecations).
+
+* It is again noted that the old scripts of the form ``osism-`` will be removed in the future.
+  A note has been added to the scripts showing this when they are executed.
+
+* The following services are deprecated and will be removed with R6 (OSISM 6.1.0):
+  * Patchman
+  * Adminer
+  * Patchman Client
+  * Virtualbmc
+  * Bird
+
+## Security Fixes
+
+Throughout the Release 5 development cycle, the SCS project issued two security advisories for upstream components:
+
+* In April 2023 an advisory in Open vSwitch (OvS) ([CVE-2023-1668](https://cve.report/CVE-2023-1668) was issued.
+Our [advisory](https://scs.community/security/2023/04/21/cve-2023-1668/).
+
+* In May 2023 an advisory affecting the OpenStack component Cinder ([CVE-2023-2088](https://cve.report/CVE-2023-2088)) was issued.
+Our [advisory](https://scs.community/security/2023/05/10/cve-2023-2088/).
+
+
+## Resolved Issues
+Numerous minor issue have been resolved. The most important steps on the IaaS side probably being the move to ceph Quincy
```

Comment:
```suggestion
Numerous minor issue have been resolved. The most important steps on the IaaS side probably being the move to ceph Quincy.
```

Reply:
No, the sentence continues after the line break.

Reply:
> No, the sentence continues after the line break.

sorry, my bad.